web-console/src/components/sql-control.tsx

```diff
@@ -218,6 +219,9 @@ export class SqlControl extends React.Component<SqlControlProps, SqlControlState
           {isRune ? 'Rune' : 'Run'}
         </Button>
         {!isRune ? SqlControlPopover : null}
+        <span className={"query-elapsed"}>
+          Last query took {queryElapsed / 1000} seconds
```

Comment:
only show this element if `queryElapsed` is defined also add a `.toFixed(2)` to format it as `4.53` right now if `queryElapsed` is 3000 this will say `Last query took 3 seconds` instead it should say `Last query took 3.00 seconds`